### ISSUE TITLE
TZNames.GetCountryNames() returns territories without time zones and different results per locale

### ISSUE THREAD

**Issue by RobJohnston** [mj1856/TimeZoneNames#53]
Calling TZNames.GetCountryNames() with the English locale gets data from https://www.unicode.org/repos/cldr/tags/release-34/common/main/en.xml, which includes 4 territories without time zones: 
* EZ - Eurozone
* UN - United Nations
* XA - Pseudo-Accents
* XB - Pseudo-Bidi

Calling it with the French locale gets data from https://www.unicode.org/repos/cldr/tags/release-34/common/main/fr.xml, which does not include "XA" nor "XB".

Expected results: 
1.  different locales return the same number of territories
2. Territories without time zones are not returned

**Comment by mattjohnsonpint** [comment]
Indeed, this is a side effect of the nature of the source data in CLDR.  We'd have to do some further analysis and manipulation of the data to resolve this.

A PR would be welcome.
